package.json

```diff
@@ -62,7 +61,7 @@
     "eslint-plugin-simple-import-sort": "^10.0.0",
     "eslint-plugin-sonarjs": "^0.23.0",
     "eslint-plugin-unicorn": "^49.0.0",
-    "husky": "4.3.8",
+    "husky": "9.0.11",
```

Comment:
required update, becuse the version 4 doesn't work well with pnpm.
for reference: https://github.com/pnpm/pnpm/issues/4790